app/Helpers/DeleteFile.php

```diff
@@ -1,14 +1,13 @@
 <?php
+
 /**
  * Created by PhpStorm.
  * User: andela
  * Date: 8/12/15
  * Time: 5:07 PM
  */
-
 namespace ChopBox\helpers;
 
-
 class DeleteFile {
 
 }
```

Comment:
I created that as a template so that it will remind me to come back to it and complete it when we have the chops upload ready. The file was intended to be a helper class for deleting images from cloudinary.